roles/openshift_metering/tasks/install.yaml

```diff
@@ -0,0 +1,93 @@
+---
+- name: Create temp directory for doing work in on target
+  command: mktemp -td openshift-metering-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+
+- name: Ensures {{ mktemp.stdout }}/metering-manifests dir exists
+  file:
+    path: "{{ mktemp.stdout }}/metering-manifests"
+    state: directory
+
+- name: Copy files to temp directory
+  copy:
+    src: "{{ item }}"
+    dest: "{{ mktemp.stdout }}/metering-manifests/{{ item | basename }}"
+  with_fileglob:
+  - "*.yaml"
+
+- name: Copy admin client config
```

Comment:
This should be unnecessary. Instead we should just reference it at `{{ openshift.common.config_base }}/master/admin.kubeconfig`